src/app/pages/ParaTimesPage/TransactionAmount/index.tsx

```diff
@@ -127,21 +152,32 @@ export const TransactionAmount = () => {
                 />
               </FormField>
               {balance && (
-                <Button
-                  disabled={disabled}
+                <Box
                   style={{
                     fontWeight: 'bold',
                     zIndex: 2,
                     position: 'absolute',
                     top: '15px',
                     right: 0,
                   }}
-                  plain
-                  label={t('paraTimes.amount.max', 'MAX')}
-                  onClick={() =>
-                    setTransactionForm({ ...transactionForm, amount: formatter(balance).replaceAll(',', '') })
-                  }
-                />
+                >
+                  <Tip
+                    content={t('paraTimes.amount.tooltip', 'Max value may be decreased by the fee')}
+                    dropProps={{ align: { bottom: 'top' } }}
+                  >
+                    <Button
```

Comment:
Doesn't work:

When withdrawing MAX `0.0055` from emerald it calculates `0.0054999999985` but fails with `Unknown gRPC error: client: transaction check failed: runtime error: module: consensus code: 5 message: consensus: amount not representable`

![err](https://user-images.githubusercontent.com/3758846/190319405-0db78daf-58ea-4bbc-94c1-96f285b7465c.gif)

Reply:
Did deposit and withdrawal amounts to any paratime need to have at most 9 decimal places? Even though EVM has 18?

Reply:
deposit and withdrawal amounts in the transaction need to be expressed in 10^-18 rose, but they have to be a multiple of 10^9 of those